lib/auth_pages/reset_password_code.dart

```diff
@@ -73,8 +68,7 @@ class _ResetPasswordCodePageState extends State<ResetPasswordCodePage> {
             builder: (context) {
               return AlertDialog(
                 title: const Text("Success!"),
-                content: const Text(
-                    "Your password has beem reset succesfully. You can now login with your new password."),
+                content: const Text("Your password has beem reset succesfully. You can now login with your new password."),
```

Comment:
been* (there's a typo here)

Reply:
Good catch, I'll fix it in my next PR (unless I end up having to publish another revision in this PR for another reason)